packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -70,18 +73,50 @@ export function useTableColumnResize<T>(props: ResizerProps<T>, state: ColumnRes
         }
         columnResizeWidthRef.current += deltaX;
         stateRef.current.onColumnResize(item, columnResizeWidthRef.current);
+        if (stateRef.current.getColumnMinWidth(item.key) >= stateRef.current.getColumnWidth(item.key)) {
+          document.body.style.setProperty('cursor', direction === 'rtl' ? 'e-resize' : 'w-resize');
+        } else if (stateRef.current.getColumnMaxWidth(item.key) <= stateRef.current.getColumnWidth(item.key)) {
+          document.body.style.setProperty('cursor', direction === 'rtl' ? 'w-resize' : 'e-resize');
+        } else {
+          document.body.style.setProperty('cursor', 'col-resize');
+        }
```

Comment:
I think the cursor directions are flipped contrary to what was stated in the description? It currently points to the right if the resizer can't be moved to the right anymore, but it should point to the left instead in that case to help the user know that they should be moving the resizer to the left in that case. Basing this behavior off of how chrome does window resizing.

 Using this story: https://reactspectrum.blob.core.windows.net/reactspectrum/39683e0659853c3ea8cf786c52d951a8b40dad3b/storybook/index.html?path=/story/tableview--allowsresizing-uncontrolled-min-max-widths&providerSwitcher-toastPosition=bottom

Reply:
Actually, maybe the code in question is in Resizer.tsx